src/Kernel/index.js

```diff
@@ -396,7 +396,11 @@ commander
     const similarCommands = Object.keys(kernel.commands).filter((c) => levenshtein.get(command, c) <= 3)
 
     console.log(`\n  error: \`${command}\` is not a registered command \n`)
-    console.log(`Did you mean ${kleur.magenta.bold(similarCommands.join(', '))} instead?`)
+
+    if (similarCommands.length >= 1) {
```

Comment:
Can be just `if (similarCommands.length)`?

Reply:
I prefer to be explicit and avoid using coercion but I can change if needed